spec.html

```diff
@@ -771,6 +771,27 @@ <h1>Implicit Completion Values</h1>
           1. Return NormalCompletion(*undefined*).
         </emu-alg>
         <p>Any reference to a Completion Record value that is in a context that does not explicitly require a complete Completion Record value is equivalent to an explicit reference to the [[Value]] field of the Completion Record value unless the Completion Record is an abrupt completion.</p>
+
+        <p>Invocations of abstract operations may be prefixed by `!` to indicate that the result will never be an abrupt completion and that the resulting Completion Record's [[Value]] field should be used in place of the return value of the operation.</p>
+        <emu-alg>
+          1. Let _val_ be ! OperationName().
```

Comment:
As in the status quo, this is an illustrative example, not a real definition. You should maybe give it a proper definition as with 'ReturnIfAbupt'.

Reply:
Since the `!` shorthand has no effect on the evaluation, I think it's a little less important to be as rigorous as we are with `ReturnIfAbrupt` expansion. If anything, the `?` shorthand would deserve that level of rigour. I would do it for `?` if the editors think that would help.